sdk/storage/azure-storage-common/inc/azure/storage/common/paged_response.hpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <type_traits>
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/raw_response.hpp>
+
+namespace Azure { namespace Storage {
```

Comment:
Keep it in the Azure namespace since that's where it will end up once moved down to core. It helps reduce the impact of the break.
```suggestion
namespace Azure {
```